hack/ccm/Makefile

```diff
@@ -0,0 +1,32 @@
+REGISTRY=quay.io/kabhat
+IMG=powervs-cloud-controller-manager
+GIT_COMMIT?=$(shell git rev-parse --short HEAD)
+TAG?=$(GIT_COMMIT)
```

Comment:
setting git commit doesn't make any sense to me, it will be good to have a version file in ccm and read from there to tag the image.

Reply:
Added tag with the format of <vpc_controller_commit_id>_<powervs_cloud_provider_commit_id>